### PR TITLE
fix(workspace): reenable a11y

### PR DESCRIPTION
**Pull request by redallen**
Fix some errors with existing components and reenable a11y tests in CI complete with a new pretty report thanks to @jessiehuff and @jschuler .

Pretty new report: https://patternfly-pr-3506-coverage.surge.sh

### CODE REVIEW

File: src/patternfly/components/Menu/examples/Menu.md [hunk 1]
```diff
@@ -556,7 +558,7 @@ import './Menu.css'
             {{/menu-item-description}}
           {{/menu-item}}
           {{#> menu-item-action}}
-            <i class="fas fa-fw fa-ellipsis-v" aria-hidden="true"></i>
+            <i class="fas fa-fw fa-ellipsis-v" aria-label="action 1"></i>
```
Comment (mcoker) [inline src/patternfly/components/Menu/examples/Menu.md:561]: I've fixed the `aria-label` issue on these examples in another PR. Can you revert these, or do you want me to suggest changes in this PR? The icon should be hidden and the label belongs on the menu-item-action.

File: src/patternfly/components/Select/examples/Select.md [hunk 2]
```diff
@@ -156,10 +150,6 @@ The single select typeahead should be used when the user is selecting one option
 ```
 
 The Dropdown Multi Select should be used when the user is selecting multiple items from a list. The user can narrow the list by typing from the keyboard. The List updates while typing. Selected items create a new chip and are removed from the list. The user may clear selections individually or all at once to restore the placeholder.
-### Accessibility
-| Attribute | Applied to | Outcome |
-| -- | -- | -- |
-| `aria-selected="true"` | `.pf-c-select__menu-item` | Should be set programmatically to indicate the active item. |
```
Comment (mcoker) [inline src/patternfly/components/Select/examples/Select.md:162]: are these no longer valid? we still set `aria-selected="true"` on  select menu items

File: src/patternfly/components/Select/select-menu-favorites.hbs [hunk 3]
```diff
@@ -73,15 +73,15 @@
...
           {{> select-menu-item-favorite-icon}}
         {{/select-menu-item}}
       {{/select-menu-wrapper}}
       {{#> select-menu-wrapper}}
         {{#> select-menu-item select-menu-item--Description="This is a description." select-menu-item--IsLink="true"}}
-            Item 5
+          Item 5
         {{/select-menu-item}}
-        {{#> select-menu-item select-menu-item--IsAction="true" select-menu-item--IsFavoriteAction="true"}}
+        {{#> select-menu-item select-menu-item--IsAction="true" select-menu-item--IsFavoriteAction="Not starred"}}
```
Comment (mcoker) [inline src/patternfly/components/Select/select-menu-favorites.hbs:84]: I've fixed this in another PR, too

File: src/patternfly/components/Select/select-menu-item.hbs [hunk 4]
```diff
@@ -1,16 +1,9 @@
...
-  {{/if}}
   {{#if select-menu-item--IsDisabled}}
     disabled
   {{/if}}
-  {{#if select-menu-wrapper--IsFavorite}}
-    {{#if select-menu-item--IsFavoriteAction}}
-      aria-label="Starred"
-    {{else}}
-      aria-label="Not starred"
-    {{/if}}
```
Comment (mcoker) [inline src/patternfly/components/Select/select-menu-item.hbs:13]: same, fixed this somewhere else

File: src/patternfly/components/Menu/examples/Menu.md (hunk 1 above)
Comment (redallen) [reply src/patternfly/components/Menu/examples/Menu.md:561]: I merged your PR, will update mine in a few mins.

File: src/patternfly/components/Select/examples/Select.md (hunk 2 above)
Comment (redallen) [reply src/patternfly/components/Select/examples/Select.md:162]: Axe was complaining that `aria-selected` is not valid on button elements. I'll double check that it was always button elements...

**Review by mcoker — APPROVED** [review]
👍

**Pull request merged**